doc/security/specs/firmware_update/index.md

```diff
@@ -18,6 +18,9 @@ The purpose of this document is to capture the process by which Tock will
 perform a complete in-place B-slot update of the running firmware and request
 that the `ROM/ROM_EXT` code boot into it.
 
+Note that this document does not cover the encrypted update process. That is
+left as a detail to be covered by the implmentation of the loader itself.
+
```

Comment:
Maybe more clearly something like "Note: for certification reasons this document does not discuss detailed guidance for encrypted updates. Additional implementation details on both regular and encrypted updates to follow when possible."

@moidx for his thoughts

Reply:
Updated copy to your suggestion.